### ISSUE TITLE
lb4 discover creates buggy code in case DB has a blob field - build fails

### ISSUE THREAD

**Issue by aaqilniz** [loopbackio/loopback-next#9043]
### Describe the bug

If there is a blob field in the database (MySQL in my case), running lb4 discover successfully creates a model but running the build script generates the following error: 

`src/models/amounts.model.ts:69:11 - error TS2304: Cannot find name 'Binary'. 69   image?: Binary; ~~~~~~`

Changing the field type to `Buffer` fixes the issue.

### Logs

```shell
npm run build

> binaryissue@0.0.1 build
> lb-tsc

src/models/amounts.model.ts:69:11 - error TS2304: Cannot find name 'Binary'.

69   image?: Binary;
             ~~~~~~

src/models/amounts.model.ts:69:11 - error TS4031: Public property 'image' of exported class has or is using private name 'Binary'.

69   image?: Binary;
             ~~~~~~


Found 2 errors in the same file, starting at: src/models/amounts.model.ts:69
```

### Reproduction

https://github.com/aaqilniz/blob-issue-lb4

**Comment by achrinza** [comment]
Thanks for opening an issue; we believe this should be a MySQL connector-specific issue. As such, we'll consolidate this issue with https://github.com/loopbackio/loopback-connector-mysql/issues/472.

**Comment by achrinza** [comment]
Duplicate of https://github.com/loopbackio/loopback-connector-mysql/issues/472

**Comment by aaqilniz** [comment]
Hi @achrinza. I opened up the issue at both repos because I was not sure where the issue should be handled. 

There was a PR in this repo that was closed without being merged. Here is the [link](https://github.com/loopbackio/loopback-next/pull/6476) to it.

Then we can change the type being returned from loopback-connector-mysql [here](https://github.com/loopbackio/loopback-connector-mysql/blob/21e3bba4c4f58435b6a15ab33c85657eec3f88b1/lib/discovery.js#L339). But changing the type here would need the change to be done to other connectors (like PostgreSQL) too.
